Gradle build fails in dev branch

- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.

Environment:
- Ubuntu 17.10 (en)
- Android Studio 3.2 Canary 11
- Gradle version 4.6
- Android SDK version 27
- Android SDK build tools 27.0.3
- Java 1.8

Stacktrace:

```
Caused by: org.gradle.internal.UncheckedException: java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource compilation failed
Output:  /home/user/StudioProjects/NewPipe/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values/values.xml:317: error: unescaped apostrophe in string
"Côted'Ivoire".
```

Proposed solution from Stack Overflow - [Error in strings.xml file in Android](https://stackoverflow.com/q/5657047/1429387).

--- a/app/src/main/res/values/settings_keys.xml
+++ b/app/src/main/res/values/settings_keys.xml
@@ -409,7 +409,7 @@
         <item>Congo, (Kinshasa)</item>
         <item>Cook Islands</item>
         <item>CostaRica</item>
-        <item>Côted'Ivoire</item>
+        <item>Côted\'Ivoire</item>
         <item>Croatia</item>
         <item>Cuba</item>
         <item>Cyprus</item>

